### DISCUSSION TITLE
Integration and analysis of multiple end points during experiment

### DISCUSSION THREAD

**Discussion by singcell** [satijalab/seurat#6911] [General]
My experiment consists of pluripotent stem cells differentiating into specific cells types which takes 10 days. I have performed scRNA-seq on cell cultures every alternate days (days 0, 2, 4, 6, 7 and 10) during differentiation. Previously, I used "[Integrating stimulated vs. control PBMC datasets](https://satijalab.org/seurat/archive/v3.2/immune_alignment.html)" vignette to integrate and compare two different conditions. However, now I have multiple days to compare each other. My goal is to identify cells lineages/types appearing or disappearing during differentiation process. What would be the best approach to perform this kind of analysis. Do I have to perform to pairwise comparison? or there is some other way.

Thank you

**Comment by Dragonmasterx87** [comment]
Hi @singcell !! 

My suggestion would be to: 
1. Integrate all of the datasets collectively without setting one-time point as a reference
2. integrate all data using the hESC/iPSC as a reference
3. Integrate all data using differentiated cells as a reference

An integrated dataset is necessary for you to use metadata and then perform DE analysis across celltypes. I would prefer example 1!
Let me know by tagging me if you need help on how to go about coding for the 3x models.

After running the 3 models see which one works for you!

Cheers,

🐲
